templates/akeneo/files/.platform.app.yaml

```diff
@@ -0,0 +1,102 @@
+runtime:
+    extensions:
+        - apcu
+        - imagick
+
+# The relationships of the application with services or other applications.
+# The left-hand side is the name of the relationship as it will be exposed
+# to the application in the environment variable. The right-hand
+# side is in the form `<service name>:<endpoint name>`.
+relationships:
+    database: "mysqldb:mysql"
+    elasticsearch: "elasticsearch:elasticsearch"
+
+# The configuration of app when it is exposed to the web.
+web:
+    locations:
+        "/":
+            root: "web"
+            passthru: "/app.php"
+
+# The size of the persistent disk of the application (in MB).
+disk: 3072
+
+# Environment and PHP variables
+variables:
+    php:
+        "date.timezone": "Europe/Paris"
+
+# The mounts that will be performed when the package is deployed.
+# Filesystem is read only by default
+mounts:
+    "var/logs":                 "shared:files/app-logs"
+    "var/cache":                "shared:files/app-cache"
+    "var/uploads":              "shared:files/app-uploads"
+    "app/file_storage":         "shared:files/app-file_storage"
+    "app/archive":              "shared:files/app-archive"
+    "app/uploads/product":      "shared:files/app-uploads"
+    "installer":                "shared:files/installer"
+    "web/media":                "shared:files/media-cache"
+
+hooks:
+    # We run build hooks before your application has been packaged.
+    build: |
+        # install nvm to use newer version of node
+        unset NPM_CONFIG_PREFIX
+        curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | dash
+        export NVM_DIR="$HOME/.nvm"
+        [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+        # install node 10 (currently latest LTS)
+        nvm install 10
+
+        # install yarn
+        npm install -g yarn
+
+        # run yarn to install frontend
+        yarn install
+        yarn run webpack
+
+        # remove dev file
+        rm web/app_dev.php
+
+    # We run deploy hook after your application has been deployed and started.
+    deploy: |
+        # clear caches
+        bin/console cache:clear --no-warmup --env=prod
+
+        # create tmp folders
+        mkdir -p /tmp/pim/file_storage
+        mkdir -p /tmp/pim/upload_tmp_dir
+
+        # run installer if no lockfile exists
+        if [ ! -f /app/installer/LOCK_INSTALL ]; then
+            bin/console pim:installer:db
+            touch /app/installer/LOCK_INSTALL
+        fi
+
+crons:
+    calculate_completeness:
+        spec: '0 23 * * *'
+        cmd: 'php /app/bin/console pim:completeness:calculate > /app/var/logs/calculate_completeness.log 2>&1'
+    refresh_versioning:
+        spec: '0 5 * * *'
+        cmd: 'php /app/bin/console pim:versioning:refresh > /app/var/logs/refresh_versioning.log 2>&1'
+    volume_aggregate:
+        spec: '0 22 * * *'
+        cmd: 'php /app/bin/console pim:volume:aggregate > /app/var/logs/volume_aggregate.log 2>&1'
+
+workers:
```

Comment:
The use of a worker guarantees that it will only work with a Medium or larger plan.  Are we OK with that?  I am if you are, but just highlighting it.

Reply:
The worker contains the worker for a queue that is used for Akeneos batch jobs (e.g. exports). We need to note this in the readme of course.